app/views/solutions/index.html.haml

```diff
@@ -47,8 +47,8 @@
           -else
             =link_to [:my, @user_solution], class: 'user-solution' do
               .person
-                %i.fa.fa-eye-slash.not-published-icon
-                =image_tag display_avatar_url(current_user, @current_user_track)
+                =graphical_icon 'eye-slash not-published-icon'
+                  =image display_avatar_url(current_user, @current_user_track), "Avatar of #{display_handle(current_user, @current_user_track)}"
```

Comment:
@kntsoriano This is a bug that stops the page from rendering. It causes a local failure on my tests. However, the tests passed on Travis. Any ideas?

Reply:
I'll try running it on my local.